barretenberg/cpp/src/barretenberg/crypto/aes128/aes128.cpp

```diff
@@ -151,7 +149,9 @@ void inverse_mix_columns(uint8_t* input)
 }
 } // namespace
 
-void expand_key(const uint8_t* key, uint8_t* round_key)
+namespace bb::crypto {
```

Comment:
Only aes128 is done this way, more are a followup but we can comment on the pattern